include/hogl/c-api/output.h

```diff
@@ -57,6 +57,30 @@ hogl_output_t hogl_new_output_stdout(hogl_format_t fmt);
  */
 hogl_output_t hogl_new_output_stderr(hogl_format_t fmt);
 
+/**
+ * File output options
+ * @note -  this is copied from output-file.hpp.
+ *          so everything that is changed in output-file.hpp needs
+ *          to be reflected also here
+ */
+struct hogl_output_options {
+	unsigned int perms;           /// File permissions
+	size_t       max_size;        /// Max size of each file chunk (bytes)
+	unsigned int max_age;         /// Max age of each file chunk (seconds)
+	unsigned int max_count;       /// Max file count. Index goes back to zero after it reaches max_count.
+	unsigned int buffer_capacity; /// Max capacity of the output buffer (bytes)
+	int          cpu;             /// The CPU number to be used
```

Comment:
Let's use cpu_set_t here. Technically the outputter can scale across multiple cores if needed (ie doesn't have to be single threaded).